### DISCUSSION TITLE
Unknown attachable member 'VirtualizingStackPanel.IsVirtualizing' on element 'TreeView'

### DISCUSSION THREAD

**Discussion by y-mk-yc** [microsoft/WinUI-Gallery#908] [Q&A]
I cannot use attached member "VirtualizingStackPanel.IsVirtualizing" not only on 'TreeView', but also some other element such as ' ListBox'. Also ,there's no wavy lines under the code. 
I cannot use NodeMouseClick, either.

![image](https://user-images.githubusercontent.com/74123687/163573751-69b04f7a-78af-4472-b44b-ddc046346132.png)
![image](https://user-images.githubusercontent.com/74123687/163665844-f744fbb9-9c21-4bab-bef6-875895308234.png)

**Comment by marcelwgn** [comment]
VirtualizingStackPanel and NodeMouseClick are WPF APIs, WinUI 3 API is based on the UWP APIs and does not support using WPF properties on WinUI 3 controls.
